### PR TITLE
[octopolo_plugin_example.gemspec] Update authors

### PR DESCRIPTION
**Pull request by NickLaMuro**
Use a more generic author name and email instead of my own.

If there is anything that you would prefer this to be instead of what I put here (meant to be a stub), feel free to let me know and I can change it, or you are welcome to update this in a different branch and close this PR as well.

### CODE REVIEW

File: octopolo-plugin-example.gemspec [hunk 1]
```diff
@@ -6,8 +6,8 @@ require 'octopolo_plugin_example/version'
 Gem::Specification.new do |spec|
   spec.name          = "octopolo-plugin-example"
   spec.version       = OctopoloPluginExample::VERSION
-  spec.authors       = ["Nick LaMuro"]
-  spec.email         = ["nick.lamuro@sportngin.com"]
+  spec.authors       = ["SportEngine's Infrastructure Engineers"]
```
Comment (emmasax4) [inline octopolo-plugin-example.gemspec:9]:
```suggestion
  spec.authors       = ["SportsEngine's Infrastructure Engineers"]
```

Comment (NickLaMuro) [reply octopolo-plugin-example.gemspec:9]: I knew I would do that...  thanks for the correction.

**Comment by emmasax4** [comment]
Thank you for the help with this.

**Comment by emmasax4** [comment]
Ah, it's possible the approval check will struggle because this is your pull request outside of the SE organization. I will just merge away.

**Comment by NickLaMuro** [comment]
> Ah, it's possible the approval check will struggle because this is your pull request outside of the SE organization.

Yeah, if I was a robot I wouldn't trust me as well.  Sounds about right.

> Thank you for the help with this.
> ...
> I will just merge away.

Thanks, and happy to help.

**Pull request merged**